### PR TITLE
Add github actions for PR metadata 

### PR DESCRIPTION
**Pull request by ggriffiths**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+area/ci:
+- .travis
+- .github/**/*
+area/osd-cmd:
+- osd/**/*
+area/api-proto:
+- api/api.pb.go
+- api/api.proto
+area/api:
+- api/*
```
Comment (harsh-px) [inline .github/labeler.yml:12]: Let's start with a wider area and keep everything under `api/*` as `area/api`.

File: .github/workflows/cron-stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    - cron: '30 1 * * *'
+
+jobs:
+  stale-in-review:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: 'This issue is stale because it has been in review for 2 days with no activity.'
```
Comment (harsh-px) [inline .github/workflows/cron-stale.yml:13]: 5 days would be a better conservative start :)

File: .github/labeler.yml (hunk 1 above)
Comment (ggriffiths) [reply .github/labeler.yml]: Ok, I kept api/server/sdk as well since I think it's helpful to know about SDK server changes. The proto regex I've removed.

Comment (ggriffiths) [reply .github/labeler.yml]: I think these labels will be even more useful in porx

**Comment by ggriffiths** [comment]
The actions will fail on this PR until they're added to the origin - they don't want people running actions on PRs from forks.

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- .travis
+- .github/**/*
+area/osd-cmd:
+- osd/**/*
+area/api:
+- api/**/*
+area/sdk-server:
+- api/server/sdk/**/*
+area/pkg:
+- pkg/**/*
```
Comment (harsh-px) [inline .github/labeler.yml:13]: the pkg area doesn't add much value since it's the kitchen sink. Can be removed unless you see a use case

**Review by harsh-px — APPROVED** [review]

File: .github/labeler.yml (hunk 3 above)
Comment (ggriffiths) [reply .github/labeler.yml]: Removed

**Pull request merged**